onyo/commands/get.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import sys
+
+from rich import box
+from rich.console import Console
+from rich.table import Table
+
+from onyo import Repo, OnyoInvalidRepoError
+
+logging.basicConfig()
+log = logging.getLogger('onyo')
+
+
+def get(args, opdir):
+    """
+    Display the requested `key`(s) in tabular form for matching assets.
+    """
+    try:
+        repo = Repo(opdir)
```

Comment:
This is default behavior for most commands, though (i.e., `cat`, `config`, `edit`, etc. all initialize the repo in a `try/except` block)

Reply:
They all have repo = None or something before that block, so the variable exists, independently of its value.

Reply:
> They all have repo = None or something before that block, so the variable exists, independently of its value.

That's not the case for all commands. A quick cursory glance shows me that `cat` and `config` don't preallocate `repo`, but `edit` and `rm` do. I will add `repo = None` before the `try/except` block in `get`, but this should probably be addressed in other commands then as well.

Reply:
I have added it; note that some linters may complain that `repo = None` is unused (which is correct), and that pyre only complains because it does not understand that the caught exception terminates the program.

Reply:
Yes, and pylint for example does complain, that every command has these repeated lines. On the long run we will probably switch to something else and make a bunch of modifications, but for now, thanks for making it consistent :)